diff --git a/dist/esm/cli.mjs b/dist/esm/cli.mjs
index ceeaa986eb3bb5452b78a206099d9005c43bc3f0..8703d9d701de1e4c19f2baa08185f4867a047941 100644
--- a/dist/esm/cli.mjs
+++ b/dist/esm/cli.mjs
@@ -23,11 +23,17 @@ function createTestingLogger() {
 	return ret;
 }
 async function ensureConfig(configPath) {
-	if (!isSupportedFormat(configPath)) {
-		throw new Error(`Unsupported config format. Expected: \`${SUPPORTED_CONFIG_FORMATS.join(",")}\` but got \`${nodePath.extname(configPath)}\``);
-	}
 	const fileUrl = (pathToFileURL(configPath)).toString();
-	const configExports = await import(fileUrl);
+	let configExports
+	try {
+		configExports = await import(fileUrl);
+	} catch (e) {
+		let errorMessage = 'Error happened while loading config.'
+		if (!isSupportedFormat(configPath)) {
+			errorMessage += ` Unsupported config format. Expected: \`${SUPPORTED_CONFIG_FORMATS.join(",")}\` but got \`${nodePath.extname(configPath)}\``
+		}
+		throw new Error(errorMessage, { cause: e })
+	}
 	return configExports.default;
 }
 const SUPPORTED_CONFIG_FORMATS = [".js", ".mjs", ".cjs"];
diff --git a/dist/types/utils/define-config.d.ts b/dist/types/utils/define-config.d.ts
index 20400ab86f6d55ecc0ef3c310bfee93edfa21524..e0df99ace07c43ae43fbb806948396cb36e3cfbb 100644
--- a/dist/types/utils/define-config.d.ts
+++ b/dist/types/utils/define-config.d.ts
@@ -1,2 +1,5 @@
+import type { RolldownOptions } from '../types/rolldown-options';
 import type { ConfigExport } from '../types/config-export';
+export declare function defineConfig(config: RolldownOptions): RolldownOptions;
+export declare function defineConfig(config: RolldownOptions[]): RolldownOptions[];
 export declare function defineConfig(config: ConfigExport): ConfigExport;
